runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/Environments.java

```diff
@@ -228,6 +232,10 @@ public static Environment createProcessEnvironment(
     return filesToStage.build();
   }
 
+  public static Set<String> getJavaCapabilities() {
+    return ModelCoders.urns();
```

Comment:
Add:
MULTI_CORE_BUNDLE_PROCESSING
PROGRESS_REPORTING

Reply:
Good call on multi-core. Is progress reporting actually done now?

Reply:
(I suppose that's just the new metrics PCollection element counts, right?)

Reply:
Yes but only some metrics are being reported.